modules/custom/openy_search/openy_google_search/openy_google_search.install

```diff
@@ -4,3 +4,106 @@
  * @file
  * Module installation file.
  */
+
+use Drupal\node\NodeInterface;
+use Drupal\pathauto\PathautoState;
+
+/**
+ * Implements hook_install().
+ */
+function openy_google_search_install() {
+
+  $alias = '/search';
+  /** @var \Drupal\Core\Path\AliasStorageInterface $path_alias_storage */
+  $path_alias_storage = \Drupal::service('path.alias_storage');
+  if ($path_alias_storage->aliasExists($alias, 'en')) {
+    return;
+  }
+  /** @var \Drupal\Core\Extension\ModuleInstallerInterface $module_installer */
+  $module_installer = \Drupal::service('module_installer');
+  if (!$module_installer->install(['openy_prgf_google_search'])) {
+    return;
+  }
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  $search_paragraph = $paragraph_storage->create([
+    'type' => 'google_search',
+    'field_prgf_block' => [
+      [
+        'plugin_id' => 'google_search_block',
+        'plugin_configuration' => [
+          'id' => 'google_search_block',
+          'label' => 'Google search',
+          'provider' => 'openy_google_search',
+          'label_display' => 'hidden',
+        ],
+      ],
+    ],
+  ]);
+  if (!$search_paragraph->save()) {
+    return;
+  }
+  // Fix broken block plugin id.
+  $tables = [
+    'paragraph__field_prgf_block',
+    'paragraph_revision__field_prgf_block',
+  ];
+  foreach ($tables as $table) {
+    if (!\Drupal::database()->schema()->tableExists($table)) {
+      continue;
+    }
+    // Select google_search paragraphs that have "broken" as plugin_id.
+    $query = \Drupal::database()->select($table, 'ptable');
+    $query->fields('ptable');
+    $query->condition('ptable.field_prgf_block_plugin_id', 'broken');
+    $query->condition('ptable.bundle', 'google_search');
+    $broken_paragraphs = $query->execute()->fetchAll();
+    foreach ($broken_paragraphs as $paragraph) {
+      $data = unserialize($paragraph->field_prgf_block_plugin_configuration);
+      $query = \Drupal::database()->update($table);
```

Comment:
@anprok Please define the database service only once before the loop and use it everywhere.